app/grandchallenge/reader_studies/models.py

```diff
@@ -1165,28 +1183,10 @@ def clean(self):
                 f"question type ({self.answer_type})"
             )
 
-    @property
-    def annotation_types(self):
-        return [
-            self.AnswerType.BOUNDING_BOX_2D,
-            self.AnswerType.MULTIPLE_2D_BOUNDING_BOXES,
-            self.AnswerType.DISTANCE_MEASUREMENT,
-            self.AnswerType.MULTIPLE_DISTANCE_MEASUREMENTS,
-            self.AnswerType.POINT,
-            self.AnswerType.MULTIPLE_POINTS,
-            self.AnswerType.POLYGON,
-            self.AnswerType.MULTIPLE_POLYGONS,
-            self.AnswerType.MASK,
-            self.AnswerType.LINE,
-            self.AnswerType.MULTIPLE_LINES,
-            self.AnswerType.ANGLE,
-            self.AnswerType.MULTIPLE_ANGLES,
-        ]
-
     @property
     def allow_null_types(self):
         return [
-            *self.annotation_types,
+            *self.AnswerType.get_annotation_types(),
```

Comment:
Does this really work? I would have thought this would result in an attribute error? `self.AnswerType.get_annotation_types()` should be `self.get_annotation_types()`. Same on L1161

Reply:
`self.get_annotatuion_types()` doesn't work because `get_annotation_types` is no longer a property or method on the `Question` model

Reply:
And yes it works, `get_annotation_types()` returns a list, and here we unpack that list. Or why did you think we would get an attribute error here?

Reply:
Ah yes, confusing construction but I see why it works now.